core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -85,6 +89,84 @@ public void register(BuiltinFunctionRepository repository) {
     repository.register(to_days());
     repository.register(week());
     repository.register(year());
+
+    repository.register(now());
```

Comment:
nit: should have inserted these alphabetically into the above list

Reply:
I'll update in next PR